lua/neogit/buffers/commit_editor/init.lua

```diff
@@ -40,24 +40,24 @@ function M:open()
     modifiable = true,
     readonly = false,
     autocmds = {
-      ["BufWritePost"] = function()
-        written = true
-      end,
       ["BufUnload"] = function()
-        if written then
-          if
-            config.values.disable_commit_confirmation
-            or input.get_confirmation("Are you sure you want to commit?")
-          then
-            vim.cmd([[
-              silent g/^#/d
-              silent w!
-            ]])
-          end
+        if
+          not config.values.disable_commit_confirmation
```

Comment:
this condition doesn't do the same thing. if you disable the commit confirmation you never enter into this condition

Reply:
> > Also, a visual feedback on cancellation would be nice - as it is done with the other events. But there doesn't seem to be a way to get a return value that can be used others with the other notification messages without doing some architetural changes. So I'm helping myself out with an autocmd.
> 
> There shouldn't be a need for visual feedback. If you say no it just doesn't do anything, so there is no async part to it.

Perfectly fine. Tell me if you like it via rebase or as commit, and I'll remove it. Or if you prefer go ahead push the changes and add yourself as co-author.

Reply:
yes that's right. but isn't it the goal that you won't get the confirmation message if you disable it?